openstates/in/bills.py

```diff
@@ -77,9 +81,127 @@ def get_name(self,random_json):
         #got sick of doing this everywhere
         return random_json["firstName"] + " " + random_json["lastName"]
 
+    def process_votes(self,rollcalls,bill,proxy):
+
+        result_types = {
+            'FAILED': False,
+            'DEFEATED': False,
+            'PREVAILED': True,
+            'PASSED': True,
+            'SUSTAINED': True,
+            'NOT SECONDED': False,
+            'OVERRIDDEN': True,
```

Comment:
This was already in the code when I got it and doesn't conflict with anything I've seen.